Formula/dvm.rb

```diff
@@ -0,0 +1,36 @@
+class Dvm < Formula
+  desc "Docker Version Manager"
+  homepage "https://github.com/getcarina/dvm"
+  version "0.6.2"
```

Comment:
Drop this line and instead inline the version into the `url` line. You will still be able to call `version` to query the version (that is auto-detected from the URL) in, e.g., `test do` or the `install` method.